<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-QBFTON6" failures="0" tests="2" name="com.mystore.testcases.End2EndTest" time="0.114" errors="1" timestamp="2025-09-23T16:41:50 IST" skipped="1">
  <testcase classname="com.mystore.testcases.End2EndTest" name="EndToEndtest" time="0.000">
    <skipped/>
  </testcase> <!-- EndToEndtest -->
  <testcase classname="com.mystore.testcases.End2EndTest" name="loadconfig" time="0.114">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at java.base/java.io.File.<init>(File.java:278)
at com.aventstack.extentreports.reporter.BasicFileReporter.init(BasicFileReporter.java:106)
at com.aventstack.extentreports.reporter.ExtentHtmlReporter.<init>(ExtentHtmlReporter.java:30)
at com.mystore.utility.ExtentManager.setExtent(ExtentManager.java:14)
at com.mystore.base.BaseClass.loadconfig(BaseClass.java:32)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:358)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- loadconfig -->
</testsuite> <!-- com.mystore.testcases.End2EndTest -->
